src/main/java/net/minecraftforge/common/extensions/IForgeEntity.java

```diff
@@ -200,4 +202,19 @@ default boolean isMultipartEntity()
     {
         return null;
     }
+
+    /**
+     * Return the height in blocks the Entity can step up without needing to jump
+     * @return The sum of vanilla's {@code maxUpStep} field and the current value
+     *  of the {@link net.minecraftforge.common.ForgeMod#STEP_HEIGHT} attribute,
+     *  (if this Entity is a {@link LivingEntity}).
+     */
+    default float getStepHeight() {
+        var vanillaStep = self().maxUpStep;
+        if (self() instanceof LivingEntity living) {
+            return (float) (vanillaStep + living.getAttributeValue(ForgeMod.STEP_HEIGHT.get()));
+        } else {
+            return vanillaStep;
+        }
+    }
```

Comment:
Formatting: braces on newlines.
```suggestion
    default float getStepHeight()
    {
        var vanillaStep = self().maxUpStep;
        if (self() instanceof LivingEntity living)
        {
            return (float) (vanillaStep + living.getAttributeValue(ForgeMod.STEP_HEIGHT.get()));
        } else
        {
            return vanillaStep;
        }
    }
```